src/pages/ReportDetailsPage.js

```diff
@@ -144,12 +158,17 @@ class ReportDetailsPage extends Component {
                                             shouldUseFullTitle={isChatRoom || isPolicyExpenseChat || isThread}
                                         />
                                     </View>
-                                    <Text
-                                        style={[styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting, styles.mb2, styles.pre]}
-                                        numberOfLines={1}
-                                    >
-                                        {chatRoomSubtitle}
-                                    </Text>
+                                    {isPolicyAdmin ? (
+                                        <Pressable
+                                            onPress={() => {
+                                                Navigation.navigate(ROUTES.getWorkspaceSettingsRoute(this.props.report.policyID));
```

Comment:
```suggestion
                                                Navigation.navigate(ROUTES.getWorkspaceInitialRoute(this.props.report.policyID));
```

I think we actually want to go to the initial workspace settings page. The one that lists all of the settings of the workspace.

Reply:
Yes, my assumption is that by "workspace settings" he means the overall workspace settings page. That's what makes the most sense. I think it's kind of confusing that we call the "general settings" page within the workspace settings `/settings`.

Reply:
I agree that we should go to workspace initial page

Reply:
All good. The language is confusing.